src/components/ItemSet/index.js

```diff
@@ -18,22 +19,32 @@ const ITEM_TYPE_COMPONENTS = {
   exoticCatalysts: MasterworkCatalyst
 };
 
-function ItemSet({
-  className,
-  inventory,
-  itemDefs,
-  setPopper,
-  setModal,
-  set,
-  objectiveInstances,
-  objectiveDefs
-}) {
-  const { name, noUi, description, sections, image } = set;
+class ItemSet extends Component{
+  constructor(props) {
+    super(props);
+    this.accordion = this.accordion.bind(this);
+  }
+  
+  accordion(){
+    const key = window.location.pathname + '-' + this.props.set.name;
+    const hidden = !ls.getAccordionState(key);
+    ls.saveAccordionState(key,hidden);
+    this.forceUpdate();
+    
+    //trigger scroll event causing lazy load to load images that should now be in view
+    setTimeout(()=> window.dispatchEvent(new Event('scroll')), 200);
```

Comment:
The lazy load library I'm using has a little utility method for this https://github.com/jasonslyvia/react-lazyload#forcecheck